oca_dependencies.txt

```diff
@@ -1,3 +1,4 @@
 account-financial-tools
 server-ux
+stock-logistics-workflow
```

Comment:
@labaggio come mai?

Reply:
ieri mi hanno segnalato un conflitto.. ho controllato e in oca_dependencies.txt  è stato aggiunto questo modulo.
vuoi che torno indietro e gestitsci tu il conflitto in fase di merge? faccio uno squash (ieri ero di fretta e ho committato con il --amend :( ) o dimmi tu.. 
però poi mergiamo?? :D

Reply:
> ho controllato e in oca_dependencies.txt è stato aggiunto questo modulo

Infatti su https://github.com/OCA/l10n-italy/blob/11.0/oca_dependencies.txt c'è già. Quindi non lo devi aggiungere anche tu.

> però poi mergiamo??

In questo momento i commit mi pare siano raddoppiati.
Se non sai il perchè, probabilmente ti conviene rifare la procedura di migrazione e aggiungere il commit di migrazione https://github.com/OCA/l10n-italy/pull/639/commits/0de57aeb6fecc25d6b8a6301176acabbd50c4af3